Xsiam sub playbooks

melamedbn opened the pull request:
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: 
https://github.com/demisto/etc/issues/46401
https://github.com/demisto/etc/issues/46390
https://github.com/demisto/etc/issues/46389
https://github.com/demisto/etc/issues/46342

## Description
The main sub-playbooks for XSIAM: Containment, eradication, investigation and recovery.

## Minimum version of Cortex XSOAR
- [x] 6.6.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

altmannyarden commented:
@jrosenpanw Can you please review at least the playbook description?

jrosenpanw commented:
@richardbluestone please could you take a look and re-assign

richardbluestone commented:
@julieschwartz18 Can you review pls?

julieschwartz18 commented:
@richardbluestone - Sure

julieschwartz18 commented:
Doc review complete for playbook-Containment_Plan.yml. @melamedbn - please confirm the following:
- Line 910 - the description for Manual endpoint isolation is "Checks whether the values provided in arguments are equal. If either of the arguments is missing, no is returned. - is this correct?
- Line 1108 - the description for Choose indicators to block is "Checks whether the values provided in arguments are equal. If either of the arguments is missing, no is returned." - is this correct?
- Line 1268 - the description for Set selected indicators to block is "Set a value in context under the key you entered." - is this correct?
- Line 1380 - the description for Set selected users for sessions clearing is "Set a value in context under the key you entered." - is this correct?
- Line 1418 - the description for Choose user to to execute clean session is "Check whether the values provided in arguments are equal. If either of the arguments are missing, no is returned." - is this correct?
- Line 1680 - the description for Choose files to quarantine is "Checks whether the values provided in arguments are equal. If either of the arguments is missing, no is returned." - is this correct?
- Line 1798 - the description for Set selected files to quarantine is "Set a value in context under the key you entered." - is this correct?
- Line 1912 - the description for Choose users to disable is "Checks whether the values provided in arguments are equal. If either of the arguments is missing, no is returned." - is this correct?
- Line 2030 - the description for Set selected users to disable is "Set a value in context under the key you entered." - is this correct?

melamedbn commented:
> Doc review complete for playbook-Containment_Plan.yml. @melamedbn - please confirm the following:
> 
> * Line 910 - the description for Manual endpoint isolation is "Checks whether the values provided in arguments are equal. If either of the arguments is missing, no is returned. - is this correct? **No, Fixed.**
> * Line 1108 - the description for Choose indicators to block is "Checks whether the values provided in arguments are equal. If either of the arguments is missing, no is returned." - is this correct? **No, Fixed.**
> * Line 1268 - the description for Set selected indicators to block is "Set a value in context under the key you entered." - is this correct? **- Yes**
> * Line 1380 - the description for Set selected users for sessions clearing is "Set a value in context under the key you entered." - is this correct? **- Yes**
> * Line 1418 - the description for Choose user to to execute clean session is "Check whether the values provided in arguments are equal. If either of the arguments are missing, no is returned." - is this correct? **No, Fixed.**
> * Line 1680 - the description for Choose files to quarantine is "Checks whether the values provided in arguments are equal. If either of the arguments is missing, no is returned." - is this correct? **No, Fixed.**
> * Line 1798 - the description for Set selected files to quarantine is "Set a value in context under the key you entered." - is this correct? **- Yes**
> * Line 1912 - the description for Choose users to disable is "Checks whether the values provided in arguments are equal. If either of the arguments is missing, no is returned." - is this correct? **No, Fixed.**
> * Line 2030 - the description for Set selected users to disable is "Set a value in context under the key you entered." - is this correct? **- Yes**

@julieschwartz18

julieschwartz18 commented:
@melamedbn - just checked the fixes and the description still needs to be updated for 
- Line 1418 - the description for Choose user to to execute clean session is "Check whether the values provided in arguments are equal. If either of the arguments are missing, no is returned."

melamedbn commented:
> @melamedbn - just checked the fixes and the description still needs to be updated for
> 
> * Line 1418 - the description for Choose user to to execute clean session is "Check whether the values provided in arguments are equal. If either of the arguments are missing, no is returned."

Thanks! fixed as well.

altmannyarden commented on Packs/CommonPlaybooks/Playbooks/playbook-Containment_Plan_README.md:
```diff
@@ -0,0 +1,56 @@
+This playbook handles all the incident containment actions available with Cortex XSIAM, including the following tasks:
```
Should replace the incident with alert

melamedbn replied:
fixed.

altmannyarden commented on Packs/CommonPlaybooks/Playbooks/playbook-Containment_Plan_README.md:
```diff
@@ -0,0 +1,56 @@
+* Block indicators (currently, the playbook supports only hashes)
+* Clear user session (currently, the playbook supports only Okta)
+
+**Note:** The playbook inputs enable manipulating the execution flow; read the input descriptions for details.
```
the execution flow or of the execution flow?

melamedbn replied:
the execution flow.

altmannyarden commented on Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Investigation_Plan_README.md:
```diff
@@ -0,0 +1,58 @@
+This playbook handles all the endpoint investigation actions available with Cortex XSIAM, including the following tasks:  
+* Pre-defined MITRE Tactics
```
I would change the description because there are no real investigation "actions" here.

I think we should mention that we are using the EDR / endpoint and insights data from multiple log source to detect / find any additional suspicious / malicious behavior  

And the investigation performs a search by:
Pre-defined MITRE Tactics and techniques
Host and attacker activity
..
...

melamedbn replied:
This playbook uses the EDR and insights data to detect abnormal and malicious activity and includes the following tasks.
fixed.
